src/pages/hardware/protocolReference/can.tsx

```diff
@@ -8,14 +8,14 @@ import { CodeSample } from "../../../components/CodeSamplesContainer";
 const integrationProcess: ReactNode = (
     <div className="paragraph">
         <p>The following process should be followed when integrating a third-party device with the GO device using our Third-Party Data CAN Protocol:</p>
-        <h2>1 - request external device id</h2>
+        <h2>1 - Request external device id</h2>
```

Comment:
```suggestion
        <h2>1 - Request external device ID</h2>
```
ID has to be all caps